### ISSUE TITLE
Badges that reference SVG files from github in a markdown file don't work in devhub

### ISSUE THREAD

**Issue by patricksimonian** [bcgov/devhub-app-web#127]
**Describe the bug**
I have a markdown file in a repository that has an image `[image](../../something.svg)`
It appears to work when i view the markdown file from Github, but in the Devhub the image is broken. 

**What it looks like**
<img width="1108" alt="screen shot 2018-11-22 at 10 43 19 am" src="https://user-images.githubusercontent.com/21046727/48918793-b57eb680-ee43-11e8-81f0-9c9e13252544.png">
**What it should look like**
<img width="1133" alt="screen shot 2018-11-22 at 10 43 39 am" src="https://user-images.githubusercontent.com/21046727/48918819-d6470c00-ee43-11e8-9e73-ecc36a935b10.png">

**Desktop (please complete the following information):**
 - OS: Any
 - Browser Any

**Smartphone (please complete the following information):**
 - Device: Any
 - OS: Any
 - Browser Any

**Comment by patricksimonian** [comment]
# The Issue
So it looks like this issue has to do with a [Github Markdown Issue](https://github.com/github/markup/issues/556#issuecomment-429215495)

## TL;DR
It appears there are two reasons why externally referenced SVG files don't work in the Devhub. 
1. Image Tags that source SVG's won't load by default (as explained in the above issue) 
> '*Referencing an SVG using an <img> tag rasterizes the image and prevents it from further loading anything. It's basically inert.*'
2. Github itself prevents external requests to its svg resources 
  - this was tested by using one of many work around for externally loaded SVG's `<object data='...'`></object>`, `<embed ...>`

## Work Around

If you are trying to load an SVG Image, perhaps provide a PNG conversion of that and reference that in your markdown instead.

If you are trying to use a **Badge** as mentioned in the example screen shots. Using the [shields.io](https://www.shields.io) API is a great alternative that ***will*** work in your markdown files as is as well as the Devhub :)

The above badge (in the screenshot) could therefor be replaced with 
`![Status: Draft](https://img.shields.io/badge/Status-Draft-blue.svg)`
![Status: Draft](https://img.shields.io/badge/Status-Draft-blue.svg)
